Automattic/sensei
# Exporter - Generate courses CSV

Part of #3049 

### Changes proposed in this Pull Request

* Add task generating CSV file for courses

### Testing instructions

* Create courses with prerequisites, modules, categories, content
* Open exporter, select `Courses`, and click `Export`
* After it's done, a prompt to download the file should pop up
* File should be in the specified exporter format, and should contain all details of the courses.
--
* Delete all courses/switch to an empty test site
* Import the exported file
* Courses should have all detail properly imported
